### PR TITLE
Fixed issue 393 autogenerated headerlinks

### PR DESCRIPTION
**Pull request by ManrajGrover**
@rxaviers sir,

I have fixed issue #393 - Use autogenerated headerlinks. Please review the pull request. I have also swapped the reference links "Build" and "Tests" under "Development" to keep things in order.

### CODE REVIEW

**Comment by rxaviers** [comment]
@ManrajGrover please, read and sign our [CLA](http://contribute.jquery.org/CLA/).

**Comment by jzaefferer** [comment]
Apart from missing CLA signature, looks good to me!

**Comment by ManrajGrover** [comment]
@rxaviers 

Okay sir, I have filled the details. May I know how to fix this?

File: README.md [hunk 1]
```diff
@@ -137,7 +132,6 @@ We do NOT embed any i18n data within our library. However, we make it really
 easy to use. Read [How to get and load CLDR JSON data](#cldr) for more
```
Comment (rxaviers) [inline README.md]: `#cldr` needs to be updated to `#2-cldr-content`. The same should be done on an occurrence of `doc/api/core/load.md`.

Comment (ManrajGrover) [inline README.md]: Okay sir. Should I another pull request?

Comment (rxaviers) [inline README.md]: Another pull request is not needed. You can simply push one more commit to this same branch.

Comment (ManrajGrover) [inline README.md]: @rxaviers Sir I see lines 137-144 also need to be fixed.Fixing.

File: README.md [hunk 2]
```diff
@@ -121,13 +118,11 @@ have changed and how to migrate older 0.x code to up-to-date 1.x.
 
 [migration guide]: doc/migrating-from-0.x.md
 
-<a name="where_to_use"></a>
 ### Where to use it?
 
 Globalize is designed to work both in the [browser](#browser_support), or in
```
Comment (rxaviers) [inline README.md]: Change `#browser_support` to `#browser-support` please.

File: README.md (hunk 1 above)
Comment (rxaviers) [inline README.md]: :+1:

File: README.md [hunk 3]
```diff
@@ -239,7 +227,6 @@ Use bower `bower install globalize`, or npm `npm install globalize`.
 1. `git clone https://github.com/jquery/globalize.git`.
 1. [Build the distribution files](https://github.com/jquery/globalize/#build).
```
Comment (rxaviers) [inline README.md]: This was previously wrong. Please, can you fix it? `https://github.com/jquery/globalize/#build` to `#build`.

**Comment by rxaviers** [comment]
Overall looks good. Please, just let us know when you push new commits.

> Okay sir, I have filled the details. May I know how to fix this?

You should be good now.

**Comment by ManrajGrover** [comment]
@rxaviers Sir made final changes. Please check.

File: README.md [hunk 4]
```diff
@@ -121,35 +118,31 @@ have changed and how to migrate older 0.x code to up-to-date 1.x.
...
-easy to use. Read [How to get and load CLDR JSON data](#cldr) for more
+easy to use. Read [How to get and load CLDR JSON data](#2-cldr-content) for more
 information on its usage.
 
-<a name="modules"></a>
 ### Pick the modules you need
 
 | File | Minified + gzipped size | Summary |
 |---|--:|---|
 | globalize.js | 1.3KB | [Core library](#core) |
```
Comment (rxaviers) [inline README.md]: #core -> #core-module

**Comment by rxaviers** [comment]
One last missing link

**Comment by ManrajGrover** [comment]
@rxaviers Sir, made the change.Sorry for missing that.

**Comment by rxaviers** [comment]
Landed. No problem for the last link. Thank you for your contribution.

**Comment by arthurvr** [comment]
There's a typo in the commit message, so the PR didn't close. Can somebody do it?

**Comment by rxaviers** [comment]
Shoot my bad. Thanks @arthurvr

**Pull request closed without merging**